plog-kafka/src/main/java/com/airbnb/plog/kafka/KafkaProvider.java

```diff
@@ -46,6 +53,18 @@ public Handler getHandler(Config config) throws Exception {
         final ProducerConfig producerConfig = new ProducerConfig(properties);
         final Producer<byte[], byte[]> producer = new Producer<byte[], byte[]>(producerConfig);
 
-        return new KafkaHandler(clientId, propagate, defaultTopic, producer);
+        EncryptionConfig encryptionConfig = new EncryptionConfig();
+        try {
+            Config encryption = config.getConfig("encryption");
+            encryptionConfig.encryptionKey = encryption.getString("key");
+            encryptionConfig.encryptionAlgorithm = encryption.getString("algorithm");
+            encryptionConfig.encryptionTransformation = encryption.getString("transformation");
+            encryptionConfig.encryptionProvider = encryption.getString("provider");
```

Comment:
same here, can you add some logging saying that we are going to use the following `key`, `algorithm`, `transformation`?